docs/tutorials/clickstream-docker.rst

```diff
@@ -315,55 +309,64 @@ Send the KSQL tables to Elasticsearch and Grafana.
         ksql>
         Exiting KSQL.
 
-2. Navigate to the tutorial directory in the Docker container:
+2. Set up the required Elasticsearch document mapping template
```

Comment:
I can't test any further on Ubuntu 18.04, since I don't have an Elasticsearch container.

Reply:
did you get chance to re-test this after my update(https://github.com/confluentinc/ksql/pull/1875#discussion_r217345050) ?

Reply:
I did, and the Elasticsearch container runs now!